### DISCUSSION TITLE
Custom Domain problem: NotServedByPagesError

### DISCUSSION THREAD

**Discussion by TengriKut1923** [community/community#132874] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Show & Tell

### Body

When I researched, there were people who said it went away on its own. But it's been going on for months.

Despite this error, the redirect works fine, but it is still annoying.

![xxx](https://github.com/user-attachments/assets/7acfb20f-cb5e-48fe-9d43-3bb2a89e5042)

**Comment by knarrff** [comment]
It seems that www.tengrikut1923.com is setup correctly, but tengrikut1923.com (without the www, as in the warning message) is not: ```dig tengrikut1923.com +noall +answer -t A``` does not return an A-record for 185.199.108.153, as it should, but instead nothing. Can you add that and see if that helps?

**Reply by TengriKut1923** [reply]
DNS record gives error from attachments.

![xxx](https://github.com/user-attachments/assets/08a5aeaa-5630-4982-8563-481681e4841e)

**Reply by knarrff** [reply]
I do not know how to add that A record to tengrikut1923.com, this is different with every DNS provider. The screenshot shows an error about being unable to add something. It does not show what it was you tried to add.

**Reply by TengriKut1923** [reply]
Here it says that A records are unnecessary. 

https://github.com/orgs/community/discussions/108530

![xxx](https://github.com/user-attachments/assets/18bd7cf9-db4e-46fc-b828-e1fdaa475328)

**Reply by knarrff** [reply]
This form seems to only allow you to change records for [something].tengrikut1923.com. I am not sure what the meaning of the "@"-character is, though. Based on the input form I would expect there to be another place that can actually set those records for the base domain tengrikut1923.com.

**Reply by TengriKut1923** [reply]
I guess A records are no longer valid.

https://github.com/orgs/community/discussions/108530#discussioncomment-8495339

**Reply by knarrff** [reply]
Scratch that: I noticed that cloudfare uses the @ as a way to know what you mean is the apex domain itself. Based on that, I would suppose this should work. If it doesn't, it might be best to ask your DNS provider how to do this on their end.

**Reply by knarrff** [reply]
> I guess A records are no longer valid.
> 
> https://github.com/orgs/community/discussions/108530#discussioncomment-8495339

Maybe they are not - but they are not causing your problem. Notice the @ entries in the thread you mentioned: you likely need at least one of those.

**Reply by TengriKut1923** [reply, marked as answer]
I forwarded the DNS servers to Cloudflare. I also entered the A records through Cloudflare, not through my DNS provider. Thanks.

![Adsız](https://github.com/user-attachments/assets/42a33c85-bbf3-4802-a40b-f8964abfe22d)
